tests/analyzer/sourcecode/eval.js

```diff
@@ -0,0 +1,3 @@
+const str = `console.log('hello world');`;
+// ruleid: eval-call
+eval(str);
```

Comment:
let's call it `test_eval_call.js`? (`test-<rulename>`) Also can we have a negative test? (`// ok: eval-call` when using a constant

Reply:
yes, I need also to change it to `eval(atob(<str>))`

Reply:
done